Source/Facades/Blackjack.Facades/UserFacade.cs

```diff
@@ -53,4 +63,12 @@ public async Task<List<UserBE>> GetUsers(string soundex)
         List<UserBE> UserBEs = userEntities?.Select(userEntity => _mapper.Map<UserBE>(userEntity)).ToList();
         return UserBEs;
     }
+
+    public async Task<List<UserBE>> UpdateUser(UserBE userBE)
+    {
+        UserEntity userEntity = _mapper.Map<UserEntity>(userBE);
+        List<UserEntity> userEntities = await _dataService.UserRepo.UpdateUser(userEntity);
```

Comment:
This has no reason to return a list of users.

It should only ever work with 1 user.